sklearn/cross_validation.py

```diff
@@ -61,6 +62,16 @@
            'train_test_split']
 
 
+class FitFailedWarning(FitFailedWarning_):
```

Comment:
This should be `_FitFailedWarning` according to your tab completion explanation before....